config.go

```diff
@@ -1717,6 +1719,20 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		cfg.Pprof.MutexProfile = cfg.MutexProfile
 	}
 
+	// Don't allow both the old dust-threshold and the new
+	// channel-max-fee-exposure to be set.
+	defaultExposure := uint64(htlcswitch.DefaultMaxFeeExposure.ToSatoshis())
+	if cfg.DustThreshold != 0 && cfg.MaxFeeExposure != defaultExposure {
```

Comment:
Q: why are we checking `cfg.MaxFeeExposure != defaultExposur` but not `cfg.MaxFeeExposure != 0`?

Reply:
I see it's set in the default, hmmm, then we'd have this issue, what if i run,
```sh
./lnd-debug --dust-threshold=100 --channel-max-fee-exposure=500000
```

Reply:
ok changed thanks for pointing this out